### PR TITLE
Add data page with details of how to use sensors.AFRICA API 

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
Fixes #163119135

## Type of change
- [X] New feature (non-breaking change which adds functionality)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
@nerdlet, quick one, do we **need** partners on this page? Even sensors stories at that. I think only support/join sensors network and footer should be included (like in join page).

**Comment by nerdlet** [comment]
@kilemensi Sure. Let me change it.

File: src/components/DataArchives/DataArchives.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  root: {
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: `flexGrow: 1`

File: src/components/DataArchives/DataArchives.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+function DataArchives({ classes }) {
+  return (
+    <Grid
+      container
+      justify="center"
+      alignItems="center"
+      className={classes.root}
+    >
+      <Grid item xs={6} justify="center" align="left" className={classes.main}>
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: Why are we using half screen only `xs={6}`? Will a small screen also be able to show all the content in half screen?

File: src/components/DataArchives/DataArchives.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        <Typography variant="h6" className={classes.typography}>
+          Sensors Data
+        </Typography>
+        <Grid item style={{ paddingTop: '3rem' }}>
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: Why a `<Grid item` inside a `<Grid item`?

File: src/components/DataArchives/DataArchives.js [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+              <li>
+                type = {type}: comma separated list of sensor types, i.e.
+                'SDS011, DHT22'.
+              </li>
+              <li>area = {area}: provides all sensors within a max radius.</li>
+              <li>
+                box = {box}: provides all sensors in a 'box' with the given
+                coordinates.
+              </li>
+              <li> country = {countryCode}: i.e. 'KE, TZ, NG, ZA, ...</li>
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: Missing `'`? ie. `'KE, TZ, NG, ZA, ...'`

File: src/components/DocumentHead/PageHeads.js [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
   HOME: '/',
   ABOUT: '/about',
+
   AIR: {
     HOME: '/air',
     ABOUT: '/air/about',
-
     // Redirect /air/city to /air/city/nairobi
     CITY_HOME: '/air/city',
+    DATA: '/air/data',
```
Comment (kilemensi) [inline src/components/DocumentHead/PageHeads.js]: Group none-city URLs together and city URLs together.

File: src/pages/air/Data.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+import Stories from '../../components/About/Stories';
+import Navbar from '../../components/Header/Navbar';
+import DataArchivesHeader from '../../components/DataArchives/DataArchivesHeader';
+
+import DataArchives from '../../components/DataArchives/DataArchives';
+import Footer from '../../components/Footer';
+import Support from '../../components/Support';
+
+const styles = theme => ({
+  aboutSupport: {
```
Comment (kilemensi) [inline src/pages/air/Data.js]: `dataSupport`?

File: src/components/DataArchives/DataArchives.js (hunk 1 above)
Comment (nerdlet) [reply src/components/DataArchives/DataArchives.js]: @kilemensi Sure...Working on it

File: src/components/DataArchives/DataArchives.js [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+              href="https://archive.sensors.africa/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              archive.sensors.africa.
+            </a>
+          </Typography>
+        </Grid>
+
+        <Typography variant="body2">
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: Move this to the bottom of **Accessing API Data**.

File: src/components/DataArchives/DataArchives.js [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+                box = {box}: provides all sensors in a 'box' with the given
+                coordinates.
+              </li>
+              <li>
+                {' '}
+                country = {countryCode}: i.e.{' '}
+                <var className={classes.var}>KE, TZ, NG, ZA, ... </var>
+              </li>
+            </ul>
+          </Typography>
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: Use `<code>` for queries also i.e. `type`, `area`, `box`, and `country`.

File: src/components/DataArchives/DataArchives.js [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+            Accessing API Data
+          </Typography>
+        </Grid>
+        <Grid>
+          <Typography variant="body2">
+            <Typography variant="body2">
+              The primary way to access sensor data is via API endpoints. There
+              are several public endpoints:
+            </Typography>
+            <p>
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: Arrange the API endpoints: All data end points and then sensor specific end points. In each section, current (now) data end point and then everything end point (if available/applicable).

File: src/components/DataArchives/DataArchives.js [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: 'white'
+  },
+  main: {
+    paddingBottom: '3rem',
+    [theme.breakpoints.up('xs')]: {
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: On small screen, you usually want content to go from left edge to right edge of the phone i.e. no left/right padding/margins.

File: src/components/DataArchives/DataArchives.js [hunk 11]
```diff
@@ -0,0 +1,202 @@
...
+      <Grid
+        item
+        xs={12}
+        sm={6}
+        justify="center"
+        align="left"
+        className={classes.main}
+      >
+        <Typography variant="h6" className={classes.typography}>
+          Sensors Data
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchives.js]: Everything on this page is talking about accessing the data, shouldn't this heading be **Access Sensors Data**?

File: src/components/DataArchives/DataArchivesHeader.js [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+      alignItems="center"
+    >
+      <Grid item xs={12}>
+        <MenuBar />
+      </Grid>
+      <Grid item xs={12}>
+        <DataArchivesHeaderContent
+          title="sensors.AFRICA DATA"
+          subheading="Sensors data is collected and uploaded to our API archives every five minutes. The API accepts the sensor data and serves as the basis for the graphical display of maps used on the site. 
+          Find out more on how to access and use this data. "
```
Comment (kilemensi) [inline src/components/DataArchives/DataArchivesHeader.js]: Is the _Find out more ...._ sentence really necessary here?

**Pull request merged**